source/index.html

```diff
@@ -26,32 +27,42 @@
   ></script>
 
 <!-- Call our scripts -->
-<script type="module" src="js/createRecipe.js"></script>
 <script type="module" src="js/main.js"> </script>
+<script type="module" src="js/createRecipe.js"></script>
 <script type="module" src="js/components/RecipeCard.js"> </script>
 <script type="module" src="js/Router.js"> </script>
 
-  <link rel="shortcut icon" href="..//favicon.ico" type="image/x-icon">
+	<link rel="shortcut icon" href="..//favicon.ico" type="image/x-icon">
+
+<!-- Site HTML: -->
   <title>Recipe Book</title>
 </head>
 <body>
-  <nav>
-    <div class="nav-logo">
-      <img src="./images/pot.svg" alt="" class="nav-icon">
-      <span class="nav-title">Recipe Book</span>
-    </div>
-    <!-- TODO: Call Router -->
-    <a href="./landing.html">Home</a>
-    <a href="./explore.html">Explore</a>
-    <a href="./saved-recipes.html">Saved Recipes</a>
-    <!-- TODO: Search bar should call Router and display search results, fetch results -->
-    <form action="./search-results.html" class="search-bar">
-      <input type="text" name="" id="" placeholder="Search recipes">
-      <button type="submit">
-        <i class="fa fa-search"></i>
-      </button>
-    </form>
-  </nav>
+	<!-- Header of website:-->
+	<section class="website-header">
```

Comment:
why are we wrapping this in a section? the `<nav>` element for the navbar is separate and outside our `<main>` tag.